public/scss/components/_breakdown-table.scss

```diff
@@ -94,4 +95,36 @@
       width: 40%;
     }
   }
+}
+
+.breakdown-table__row--answers {
+  &:first-child {
+    border-top: 0;
+  }
+  
+  @include md {
+    border-left: 0;
+    border-top: 1px solid $color-grey-light;
+  }
+  .breakdown-table__row-key {
+    font-weight: 700;
+
+    @include md {
+      width: 55%;
+    }
+  }
+
+  .breakdown-table__row-value {
+    text-align: left;
+
+    @include md {
+      width: 35%;
+    }
+  }
+
+  .breakdown-table__row-link {
+    @include md {
+      width: 20%;
+    }
```

Comment:
Let's bump the links down on mobile. 

Like this https://design-system.service.gov.uk/components/summary-list/default/index.html